src/langapi/mode.cpp

```diff
@@ -103,30 +134,25 @@ std::unique_ptr<languaget> get_language_from_filename(
 {
   std::size_t ext_pos=filename.rfind('.');
 
-  if(ext_pos==std::string::npos)
+  if(ext_pos == std::string::npos)
     return nullptr;
 
   std::string extension=
     std::string(filename, ext_pos+1, std::string::npos);
 
-  if(extension=="")
+  if(extension.empty())
```

Comment:
Random fixes makes this long commit even harder to review - please avoid in future :slightly_smiling_face: